config/services_test.yaml

```diff
@@ -32,6 +27,10 @@ services:
       arguments:
         - '@messenger.bus.query'
 
+    App\Infrastructure\Share\Bus\EventBus:
```

Comment:
No need to define this here. It's extending the services.yaml so already present